package.json

```diff
@@ -13,63 +13,62 @@
     "lint:fix": "npm run lint -- --fix"
   },
   "devDependencies": {
-    "@aws-cdk/aws-apigatewayv2-alpha": "^2.18.0-alpha.0",
-    "@aws-cdk/aws-appsync-alpha": "^2.18.0-alpha.0",
-    "@aws-cdk/aws-redshift-alpha": "^2.18.0-alpha.0",
-    "@aws-cdk/aws-synthetics-alpha": "^2.18.0-alpha.0",
-    "@types/aws-lambda": "^8.10.110",
+    "@aws-cdk/aws-apigatewayv2-alpha": "^2.65.0-alpha.0",
+    "@aws-cdk/aws-redshift-alpha": "^2.65.0-alpha.0",
+    "@aws-cdk/aws-synthetics-alpha": "^2.65.0-alpha.0",
+    "@types/aws-lambda": "^8.10.114",
     "@types/http-errors": "^2.0.1",
-    "@types/jest": "^29.4.0",
-    "@types/lodash": "^4.14.191",
-    "@types/node": "^18.13.0",
+    "@types/jest": "^29.5.0",
+    "@types/lodash": "^4.14.192",
+    "@types/node": "^18.15.11",
     "@types/puppeteer": "^7.0.4",
     "@types/sax": "^1.2.4",
-    "@typescript-eslint/eslint-plugin": "^5.52.0",
-    "@typescript-eslint/parser": "^5.52.0",
-    "aws-cdk": "2.67.0",
-    "aws-sdk-client-mock": "^2.0.1",
-    "esbuild": "^0.17.8",
-    "eslint": "^8.34.0",
-    "eslint-config-prettier": "^8.6.0",
+    "@typescript-eslint/eslint-plugin": "^5.57.1",
+    "@typescript-eslint/parser": "^5.57.1",
+    "aws-cdk": "2.73.0",
+    "aws-sdk-client-mock": "^2.1.1",
+    "esbuild": "^0.17.15",
+    "eslint": "^8.37.0",
+    "eslint-config-prettier": "^8.8.0",
     "eslint-config-standard": "^17.0.0",
-    "eslint-import-resolver-typescript": "^3.5.3",
+    "eslint-import-resolver-typescript": "^3.5.5",
     "eslint-plugin-ava": "^14.0.0",
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-json": "^3.1.0",
-    "eslint-plugin-n": "^15.6.1",
+    "eslint-plugin-n": "^15.7.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-promise": "^6.1.1",
-    "jest": "^29.4.3",
-    "prettier": "^2.8.4",
-    "ts-jest": "^29.0.5",
+    "jest": "^29.5.0",
+    "prettier": "^2.8.7",
+    "ts-jest": "^29.1.0",
     "ts-node": "^10.9.1",
-    "typescript": "~4.9.5"
+    "typescript": "~5.0.3"
```

Comment:
This is major version upgrade.  I wonder what the potential impact is.  Maybe it's no big deal.